daprdocs/content/en/developing-applications/building-blocks/workflow/workflow-architecture.md

```diff
@@ -145,6 +145,8 @@ Different state store implementations may implicitly put restrictions on the typ
 
 Similarly, if a state store imposes restrictions on the size of a batch transaction, that may limit the number of parallel actions that can be scheduled by a workflow.
 
+Workflow state can be purged from a state store, purging all its history. Purging removes all metadata related to a specific workflow instance.
```

Comment:
Here's a suggestion for how to rephrase this.

```suggestion
Workflow state can be purged from a state store, including all its history. Each Dapr SDK exposes APIs for purging all metadata related to specific workflow instances.
```

Reply:
Looks good.